Swabbr.Api/Startup.cs

```diff
@@ -230,10 +230,11 @@ private void SetupIdentity(IServiceCollection services)
             // Add Identity middleware
             services.AddIdentity<SwabbrIdentityUser, SwabbrIdentityRole>(setup =>
             {
-                setup.Password.RequireDigit = true;
-                setup.Password.RequireUppercase = true;
-                setup.Password.RequireLowercase = true;
-                setup.Password.RequireNonAlphanumeric = true;
+                // TODO Cleanup
+                //setup.Password.RequireDigit = true;
+                //setup.Password.RequireUppercase = true;
+                //setup.Password.RequireLowercase = true;
+                //setup.Password.RequireNonAlphanumeric = true;
```

Comment:
Should probably just remove this instead of comment?

Reply:
Agreed, leaving the todo